CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.1.7 - February 06, 2020
```

Comment:
What about 1.1.4 - 1.1.6? They should still be in the changelog

Reply:
What should I say? Testing 1.1.4-1.1.6?